templates/common-configmap.yaml

```diff
@@ -19,8 +19,7 @@ data:
   NEO4J_dbms_connector_bolt_listen__address: 0.0.0.0:7687
   NEO4J_dbms_connector_http_listen__address: 0.0.0.0:7474
   NEO4J_dbms_connector_https_listen__address: 0.0.0.0:7473
-  NEO4J_causal__clustering_minimum__core__cluster__size__at__formation: "3"
-  NEO4J_causal__clustering_minimum__core__cluster__size__at__runtime: "3"
+  NEO4J_causal__clustering_minimum__core__cluster__size__at__runtime: "2"
```

Comment:
Sorry bit of confusion here. We should remove `causal__clustering_minimum__core__cluster__size__at__runtime:` as well.

Setting it to 2 by default is not desirable:
see https://neo4j.com/developer/kb/a-demonstration-of-cluster-size-at-formation-and-runtime/ 

Setting it to 3 is just re-stating the default value that Neo4j has and hard coding it to 3 here causes confusion if users of the helm chart want to change it.